classes/Is.php

```diff
@@ -49,24 +53,41 @@ public static function accessible( $who = '', $roles = array(), $args = array()
 
 			case 'logged_in':
 				if ( ! $logged_in ) {
-					$exclude = true;
-				} elseif ( ! empty( $roles ) ) {
-
-					// Checks all roles, should not exclude if any are active.
-					$valid_role = false;
-
-					foreach ( $roles as $role ) {
-						if ( current_user_can( $role ) ) {
-							$valid_role = true;
-							break;
-						}
-					}
-
-					if ( ! $valid_role ) {
-						$exclude = true;
-					}
-				}
-				break;
+									$exclude = true;
+								} elseif ( ! empty( $roles ) || ! empty ($userids) ) {
```

Comment:
Just split it into 2 separate elseif statements, make user_ids come first to prioritize them over roles.